components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/KeyRotationService.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.keyrotation;
+
+import org.wso2.carbon.identity.keyrotation.config.KeyRotationConfig;
+import org.wso2.carbon.identity.keyrotation.service.ConfigFileKeyRotator;
+import org.wso2.carbon.identity.keyrotation.service.DBKeyRotator;
+import org.wso2.carbon.identity.keyrotation.service.SyncedDataKeyRotator;
+import org.wso2.carbon.identity.keyrotation.util.KeyRotationException;
+
+/**
+ * Class that calls the symmetric key rotation service.
+ */
+public class KeyRotationService {
+
+    private static final String TRUE = "true";
+
+    public static void main(String[] args) throws KeyRotationException {
+
+        KeyRotationConfig configs = KeyRotationConfig.getInstance().loadConfigs(args);
+        if (TRUE.equals(configs.getEnableDBMigrator())) {
```

Comment:
Can't we use the boolean functions here? so that we don't need to define the a new property

Reply:
Addressed with https://github.com/wso2/identity-tools/pull/13/commits/c1822aa3638f426a55388883199f2d5788aa91ae